constraint/pkg/client/drivers/k8scel/schema/schema.go

```diff
@@ -25,18 +27,23 @@ var (
 )
 
 type Validation struct {
-	// A CEL expression. Maps to ValidationAdmissionPolicy's spec.validations.expression
-	Expression        string
-	Message           string
-	MessageExpression string
+	// A CEL expression. Maps to ValidationAdmissionPolicy's spec.validations
```

Comment:
I think this comment is meant for the `Validation` type so might be good to move it before the type definition.